core/domain/collection_services_test.py

```diff
@@ -626,6 +626,8 @@ def test_create_new_collection_error_cases(self):
         with self.assertRaisesRegexp(Exception, 'between 1 and 50 characters'):
             collection_services.save_new_collection(self.OWNER_ID, collection)
 
+        collection = self.save_new_valid_collection
```

Comment:
Is this right? I think you meant to call ``save_new_valid_collection()``, except I'm not sure whether you would want to here. Please remove if it doesn't have an intended side effect.